pulsar-broker/src/test/java/org/apache/pulsar/broker/resourcegroup/RGUsageMTAggrWaitForAllMsgsTest.java

```diff
@@ -164,22 +165,22 @@ public void run() {
                     sentNumBytes += mesg.length;
                     sentNumMsgs++;
                     log.debug("Producer={}, sent msg-ix={}, msgId={}", producerId, ix, msgId);
-                } catch (PulsarClientException p) {
+                } catch (PulsarClientException e) {
                     numExceptions++;
-                    log.info("Producer={} got exception while sending {}-th time: ex={}",
-                            producerId, ix, p.getMessage());
+                    log.error("Producer={} got exception while sending {}-th time: ex={}",
+                            producerId, ix, e.getMessage());
                 }
             }
             try {
                 producer.flush();
                 producer.close();
-            } catch (PulsarClientException p) {
+            } catch (PulsarClientException e) {
                 numExceptions++;
-                log.info("Producer={} got exception while closing producer: ex={}",
-                        producerId, p.getMessage());
+                log.error("Producer={} got exception while closing producer: ex={}",
+                        producerId, e.getMessage());
             }
 
-            log.debug("Producer={} done with topic={}; got {} exceptions", producerId, myProduceTopic, numExceptions);
+            log.info("Producer={} done with topic={}; got {} exceptions", producerId, myProduceTopic, numExceptions);
```

Comment:
These changes seem to only change the logging level. @liudezhi2098 Do you think that this makes a difference and resolves the flakiness in the test?

Reply:
Mainly to add a wait during setup()，logging level just a matter of better positioning.

Reply:
ok, now I see other changes.